app/grants/templates/grants/detail.html

```diff
@@ -377,4 +412,10 @@ <h5 class="font-body mt-2 font-weight-semibold">
   <script src="{% static "v2/js/grants/shared.js" %}"></script>
   <script src="{% static "v2/js/grants/detail.js" %}"></script>
   <script src="{% static "v2/js/waiting_room_entertainment.js" %}"></script>
+  <script src="{% static "v2/js/pages/tabs.js" %}"></script>
+  <script>
+    $(document).ready(function() {
+      setupTabs('#grant-detail-tabs');
+    });
+  </script>  
```

Comment:
Could we shift this into detail.js ? 
Also arrow functions ^_^

Reply:
ok.